kubernetes/riot.yml.erb

```diff
@@ -0,0 +1,121 @@
+---
+apiVersion: v1
+data:
+  config.json: |
+    {
+        "default_server_config": {
+            "m.homeserver": {
+                "base_url": "https://matrix.ocf.berkeley.edu",
+                "server_name": "matrix.ocf.berkeley.edu"
+            },
+            "m.identity_server": {
+                "base_url": "https://matrix.ocf.berkeley.edu"
+            }
+        },
+        "disable_identity_server": false,
+        "disable_custom_urls": false,
+        "disable_guests": true,
+        "disable_login_language_selector": false,
+        "disable_3pid_login": false,
+        "brand": "OCF",
+        "integrations_ui_url": "https://scalar.vector.im/",
+        "integrations_rest_url": "https://scalar.vector.im/api",
+        "integrations_jitsi_widget_url": "https://scalar.vector.im/api/widgets/jitsi.html",
+        "bug_report_endpoint_url": "https://riot.im/bugreports/submit",
+        "defaultCountryCode": "GB",
+        "showLabsSettings": true,
+        "features": {
+            "feature_pinning": "labs",
+            "feature_custom_status": "labs",
+            "feature_custom_tags": "labs",
+            "feature_state_counters": "labs",
+            "feature_many_integration_managers": "labs",
+            "feature_mjolnir": "labs",
+            "feature_dm_verification": "labs",
+            "feature_cross_signing": "labs"
+        },
+        "default_federate": true,
+        "welcomePageUrl": "home.html",
+        "default_theme": "dark",
+        "roomDirectory": {
+            "servers": [
+                "matrix.ocf.berkeley.edu",
+                "matrix.org"
+            ]
+        },
+        "enable_presence_by_hs_url": {
+            "https://matrix.org": false
+        }
+    }
+kind: ConfigMap
+metadata:
+  labels:
+    app: riot
+  name: riot-config
+  namespace: matrix
```

Comment:
here and elsewhere in the PR: leave out the namespace field, as it will be set by deploy tools, and this gives us more flexibility. Also, this stuff should be in the app-matrix namespace.